tests/test_infer/test_ops/cuda/test_get_cos_and_sin.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+import pytest
+import torch
+
+from colossalai.kernel.kernel_loader import InferenceOpsLoader
+from tests.test_infer.test_ops.triton.test_xine_copy import get_cos_sin
+
+inference_ops = InferenceOpsLoader().load()
+
+
+def numpy_equal(x, y):
+    x_numpy = x.detach().cpu().numpy()
+    y_numpy = y.detach().cpu().numpy()
+
+    np.testing.assert_allclose(x_numpy, y_numpy)
```

Comment:
This seems to be `numpy_allclose`

Reply:
Is it possible to use metric of "equivalence" to compare kernels of getting cos/sin cache?

Reply:
Done